packaging/Microsoft.NET.Sdk.WindowsDesktop/targets/Microsoft.NET.Sdk.WindowsDesktop.props

```diff
@@ -161,4 +161,11 @@
     <SupportedTargetFramework Remove="@(_UnsupportedNETCoreAppTargetFramework);@(_UnsupportedNETStandardTargetFramework);@(_UnsupportedNETFrameworkTargetFramework)" />
   </ItemGroup>
 
+  <!--
+    Import Windows Forms props.
+    These come via the Windows Forms transport package, that can be found under
+    https://github.com/dotnet/winforms/tree/main/pkg/Microsoft.Private.Winforms/sdk
+  -->
+  <Import Project="Microsoft.NET.SDk.WindowsDesktop.WindowsForms.props" />
```

Comment:
```suggestion
  <Import Project="Microsoft.NET.Sdk.WindowsDesktop.WindowsForms.props"/>
```

Case fix

Reply:
Thank you

Reply:
Getting it fixed in our transport package too https://github.com/dotnet/winforms/pull/5332